lib/Document.js

```diff
@@ -398,6 +403,7 @@ function DocumentCarrier(model) {
 			Document[key] = Document.Model[key].bind(Document.Model);
 		}
 	});
+	Document.serializer = model.serializer;
```

Comment:
We should probably put this under the `Document.prototype.serialize = function(...args) {` method above.

Reply:
This isn't actually necessary, just the injection of the Document.serialize() function. Will fix this.